docs/databases/swift-kuery.html

```diff
@@ -132,20 +141,21 @@ <h3 class="heading-3">Linux</h3>
           <h2 class="heading-2"><span class="blue-text">Step 2b:</span> Create a connection to an SQLite database</h2>
           <p>First we need to import the SwiftKuery and SwiftKuerySQLite packages.</p>
           <pre><code class="language-swift">import SwiftKuery
-  import SwiftKuerySQLite</code></pre>
-          <p>Next we create a connection pool of connections:</p>
-          <pre><code class="language-swift">let pool = SQLiteConnection.createPool(filename: "myDB.db", poolOptions: ConnectionPoolOptions(initialCapacity: 10, maxCapacity: 50))</code></pre>
+import SwiftKuerySQLite</code></pre>
+          <p>Inside the `App` extension, create a connection pool of connections:</p>
+          <pre><code class="language-swift">let pool = SQLiteConnection.createPool(filename: "myDB.db", poolOptions: ConnectionPoolOptions(initialCapacity: 10, maxCapacity: 50))
+// Create table instance here</code></pre>
           <p>This creates a pool of connections for us to use to make requests to our database.</p>
         </div>
       </section>
       <h2 class="heading-2"><span class="blue-text">Step 3:</span> Create a class to represent the database table</h2>
       <p>To use SwiftKuery we need to create a class that matches our table.</p>
-      <p>Create a new file called `Titles.swift` in `Sources` > `Application` > `Models`.</p>
-      <p>Inside `Titles.swift`, Define your Titles class:</p>
+      <p>Create a new file called `BookTable.swift` in `Sources` > `Application` > `Models`.</p>
```

Comment:
You need to explain how this relates to the Book model in `Book.swift` which we created previously in the Models directory.